minkasi/maps/mapset.py

```diff
@@ -230,6 +233,38 @@ def mpi_reduce(self):
             for map in self.maps:
                 map.mpi_reduce()
 
+class PriorMapset(Mapset):
+    """
+    Class to store and operate on WavSkyMaps. WavSkyMaps can be stored in Mapset but this
+    class is needed to apply priors.
+    
+    Attributes
+    ----------
+    nmap : int
+        Number of maps in the set.
+    maps : list[WavSkyMap]
+        
+    """
+    priormaps: List[MapType]
```

Comment:
Shouldn't this specifically be `WavSkyMap`? Also this variable doesn't exist. Did you mean to overwrite the hint on `maps`?

Reply:
Yeah it should be. I'm honestly not sure what's up with priormaps. I think this is copied from MapSetTwoRes, with a change to the variable name.